README.md

```diff
@@ -560,39 +560,52 @@ Shows the pdf `document` from the local device filesystem, or your app's assets.
 - cmake
 - npm
 - PSPDFKit for Windows.vsix (installed)
+- Powershell
 
 #### Getting Started
 
 Let's create a simple app that integrates PSPDFKit and uses the react-native-pspdfkit module.
 
-1. Make sure `react-native-cli` is installed: `npm install -g react-native-cli`.
-2. Install the windows helper plugin: `npm install --save-dev rnpm-plugin-windows`.
-3. Create the app with `react-native init YourApp`.
-4. Step into your newly created app folder: `cd YourApp`.
-5. Install windows tool for react native: `npm install --global --production windows-build-tools`.
-6. Install `react-native-pspdfkit` from GitHub: `npm add github:PSPDFKit/react-native`.
-7. Install all modules: `npm install`.
-8. Link module `react-native-pspdfkit`: `react-native link react-native-pspdfkit`.
-9. Initialise the windows project: `react-native windows`.
-10. Open the windows solution in `react-native\YourApp\windows`.
-11. Accept and install any required extensions when prompted.
-12. If the settings windows opens, click on `Developer` and select `yes`.
-13. Add `react-native-pspdfkit` to your soloution: Right click on the solution -> Add -> Existing Project. Navigate to `node_modules/react-native-pspdfkit/windows/ReactNativePSPDFKit/ReactNativePSPDFKit/` and select the `ReactNativePSPDFKit.csproj`
+1. Open `Powershell` as administrator.
+2. Make sure `react-native-cli` is installed: `npm install -g react-native-cli`.
+3. Install windows tool for react native: `npm install --global --production windows-build-tools`.
```

Comment:
"windows tool for react native" should be capitalized: "Windows Tool for React Native"